tsconfig.json

```diff
@@ -30,5 +30,5 @@
     }
   },
   "include": ["packages"],
-  "exclude": ["**/node_modules", "**/dist"]
+  "exclude": ["**/node_modules", "**/dist", "**/tests", "**/test"]
```

Comment:
I had to had the tests to the tsconfig exclude because I was getting tons of compilation errors, maybe after #3010  the errors will be gone

Reply:
![image](https://user-images.githubusercontent.com/8672915/120909174-61720100-c640-11eb-95ce-d25806b5b6dd.png)

Reply:
That's why we have tsconfig.build.json to exclude the files we don't want to compile. If you exclude test files in the base config file we won't be able to benefit from the typechecking in the test files like paths, the same rules etc etc